contracts/contracts/rem.attr/src/rem.attr.cpp

```diff
@@ -62,13 +62,13 @@ namespace eosio {
    {
       require_auth( issuer );
       require_recipient( receiver );
-      check( !value.empty(), "value is empty" );
 
       attribute_info_table attributes_info( _self, _self.value );
       const auto& attrinfo = attributes_info.get( attribute_name.value, "attribute does not exist" );
       check(attrinfo.next_id < std::numeric_limits<uint64_t>::max(), "attribute storage is full");
       check( attrinfo.is_valid(), "this attribute is beeing deleted" );
       check_permission(issuer, receiver, attrinfo.ptype);
+      check_attribute_data(value, attrinfo.type);
```

Comment:
Add please tests for this function.

Reply:
it's not an interface, just utility function for `setattr`, I don't think we need a dedicated test for it.